### PR TITLE
Preprod: movie: Encounter (2021) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Rating 3/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2020s.st.txt [hunk 1]
```diff
@@ -938,3 +938,47 @@ human vs. beast [Paul and Lady Jessica advanced across the desert sands with utm
...
+
+:: References
+https://en.wikipedia.org/wiki/Encounter_(2021_film)
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+father and son [The story explores how the boy Jay comes to understand that his estranged father is not the hero that he'd believed him to be.]
+child abduction [This is a story about well-intentioned, but misguided father abducting his two young sons to protect them from a threat that exists solely in his imagination.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2020s.st.txt:966]: Missing indefinite article before "well-intentioned"

**Review by odinlake — APPROVED** [review]
A typo noted

File: notes/stories/film/film-scifi-2020s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2020s.st.txt]: Indeed. Fixed and committed.

**Comment by odinlake** [comment]
+child abduction [This is a story about well-intentioned, but misguided*,*  father abducting his two young sons to protect them from a threat that exists solely in his imagination.]

missing comma after "misguided"

**Pull request merged**